ui/src/pages/ActionsToolbar.js

```diff
@@ -198,51 +198,51 @@ function PlayButtons() {
   return (
     <>
       <OptionsButton
-        disabled={playableDevCardTypes.size === 0 || isPlayingYearOfPlenty}
+        disabled={playableDevCardTypes.size === 0 || isPlayingMonopoly || isPlayingYearOfPlenty || isRoadBuilding}
         menuListId="use-menu-list"
         icon={<SimCardIcon />}
         items={useItems}
       >
         Use
       </OptionsButton>
       <OptionsButton
-        disabled={buildActionTypes.size === 0 || isPlayingYearOfPlenty}
+        disabled={buildActionTypes.size === 0 || isPlayingMonopoly || isPlayingYearOfPlenty || isRoadBuilding}
         menuListId="build-menu-list"
         icon={<BuildIcon />}
         items={buildItems}
       >
         Buy
       </OptionsButton>
       <OptionsButton
-        disabled={tradeItems.length === 0 || isPlayingYearOfPlenty}
+        disabled={tradeItems.length === 0 || isPlayingMonopoly || isPlayingYearOfPlenty || isRoadBuilding}
```

Comment:
NIT: I'd be awesome if we can give the `isPlayingMonopoly || isPlayingYearOfPlenty || isRoadBuilding` expression a name. Can you think of one? I can't 😅 . If not, that's ok, we don't have to force it, as is its good.